components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/introspection/IntrospectionResponseBuilder.java

```diff
@@ -78,6 +78,18 @@ public IntrospectionResponseBuilder setIssuedAt(long issuedAt) {
         return this;
     }
 
+    /**
+     * Add the ACR attribute into the introspection response if it's available.
+     * @param acr acr string
+     * @return IntrospectionResponseBuilder
+     */
+    public IntrospectionResponseBuilder setAcr(String acr) {
+        if (StringUtils.isNotBlank(acr)) {
```

Comment:
```suggestion

        if (StringUtils.isNotBlank(acr)) {
```

Let's keep it consistent with other methods